infra/modules/setup/aws_auth.tf

```diff
@@ -20,7 +20,12 @@ resource "kubernetes_config_map" "aws_auth" {
   groups:
     - system:masters
 EOF
-
+    mapUsers = <<EOF
+- userarn: ${var.user_role_arn}
+  username: dev-eks-deployer
+  groups:
+    - system:masters
```

Comment:
This section should be optional, and only be set when the user name / arn variables are set.